### ISSUE TITLE
Add an option to xpsimfmt to only keep the columns in the Level-2 files

### ISSUE THREAD

**Issue by lucabaldini** [lucabaldini/ixpeobssim#685]
This will come handy to distribute simulated event files for the purpose of testing polarization analysis, where one doesn't need anything more than the content of Level 2 files.

**Comment by lucabaldini** [comment]
I am looking into this right now. The input files produced by plain ixpesim (i.e., without the photon list mechanism) has three binary extensions:

* `EVENTS` (34 columns)
* `GTI`
* `MONTE CARLO` (14 columns)

I think the baseline might be to limit the default of the `EVENTS` extension to the content of the Level 2 files produced by the SOC, and limit the content of the `MONTE_CARLO` extension to the true energy.

**Comment by lucabaldini** [comment]
Also, a quick look at the file sizes with the current version of the xpsimfmt app:

```
-rw-r--r--. 1 lbaldini lbaldini 103M Nov  2 19:27 /data/work/ixpe/irfdata/allx_650mbar_r12345_recon.fits
-rw-r--r--. 1 lbaldini lbaldini 118M Jan 23 09:44 /data/work/ixpe/irfdata/allx_650mbar_r12345_recon_simfmt.fits
[500 k events]

-rw-r--r--. 1 lbaldini lbaldini  11M Nov  2 19:31 /data/work/ixpe/irfdata/line_10.00keV_600mbar_r54321_recon.fits
-rw-r--r--. 1 lbaldini lbaldini  12M Jan 18 10:25 /data/work/ixpe/irfdata/line_10.00keV_600mbar_r54321_recon_simfmt.fits
[50 k events]
```

Essentially we're talking about 200 MB for 1M events

**Comment by lucabaldini** [comment]
Looking more at the columns that we want to keep, I think we have two main use cases:

* IRF generation, where we essentially use  `mc_cols = ('ENERGY', 'ABS_Z', 'PE_PHI')` and `rec_cols = ('PHA', 'DETPHI1', 'DETPHI2', 'TRK_M2L', 'TRK_M2T')`
* scientific analysis, where we only care about what's included in the Level-2 files: `TRG_ID, TIME, STATUS, STATUS2, PI, W_MOM, X, Y, Q, U`

**Comment by lucabaldini** [comment]
Although I do have to say that, at least as far as the response files are concerned, we might want to keep a few extra columns to support future additions to ixpeobssim---`ROI_SIZE, NUM_PIX come to mind, but we should probably look into the background subtraction paper to make sure we don't have to regenerate files countless times.

**Comment by lucabaldini** [comment]
I have a first prototype in place that is more or less ready to be tested.

**Comment by lucabaldini** [comment]
Shipped in 30.1.0
